source/common/secret/sds_api.h

```diff
@@ -32,11 +32,21 @@ namespace Secret {
  */
 class SdsApi : public Config::SubscriptionCallbacks {
 public:
+  struct SecretData {
```

Comment:
rename to something like ```SecretMetadata``` ?

Reply:
i'm following the same naming in cluster_manager_impl, where they use clusterdata.